doc/tools/print_keys.py

```diff
@@ -3,21 +3,37 @@
 You can use this tool to find out values of keypresses
 """
 
+# pylint: disable=import-error,wrong-import-position
+
 from __future__ import (absolute_import, division, print_function)
 
 import curses
+import os
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
```

Comment:
Why do we need to mess with the PYTHONPATH here?

Reply:
This is used to import `ranger.ext.keybinding_parser` without needing to install `ranger`. This file can be run directly with:

```bash
python doc/tools/print_keys.py
```

Reply:
I understand what the code does but this is a bad way of doing it and won't be accepted.

Reply:
I resolved this.